apps/web/src/styles/globals.css

```
@@ -149,3 +149,175 @@
+  height: 14px;
+  background: transparent;
+  border: none;
+}
+
+.spread-slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)